src/pages/tasks/kanban/Kanban.tsx

```diff
@@ -324,97 +332,117 @@ export function Kanban() {
       </div>
 
       {board && (
-        <div
-          className="flex pb-6 space-x-4 overflow-x-auto mt-4"
-          style={{ paddingRight: isKanbanViewSliderVisible ? 512 : 0 }}
-        >
-          <DragDropContext onDragEnd={onDragEnd}>
-            {board.columns.map((board) => (
-              <Droppable key={board.id} droppableId={board.id}>
-                {(provided) => (
-                  <div
-                    className="bg-white rounded shadow select-none h-max"
-                    style={{ minWidth: 360 }}
-                  >
-                    <div className="border-b border-gray-200 px-4 py-5">
-                      <h3 className="leading-6 font-medium text-gray-900">
-                        {board.title}
-                      </h3>
-                    </div>
-
```

Comment:
Can you explain why we are wrapping in double flex?

Reply:
If I just put a new icon div in a div with classes `flex pb-6 space-x-4 overflow-x-auto mt-4` and if I say ok be sticky and `right :0` it will look like this:

![Screenshot 2023-01-17 at 14 42 58](https://user-images.githubusercontent.com/51542191/212915647-bda4d324-b982-4cef-aad1-7fc14fb11657.png)

The icon div doesn't know that we have empty space on the right because of the padding. So we need one parent to run this part. In that case the div with the mentioned classes will be styled with `pr-6` and the icon will fit without any problems. If I exclude the flex class from the div I just added, the icon will be at the bottom (direction will be column by default).

I've only tried some other solutions, but I haven't found any simpler than this one.

Reply:
As long as the icon appears next to the last status, i'm fine.